src/processes.c

```diff
@@ -1429,6 +1429,41 @@ static int ps_read_process(long pid, process_entry_t *ps, char *state) {
   return 0;
 } /* int ps_read_process (...) */
 
+static int procs_running(void) {
+  char buffer[4096] = {};
```

Comment:
4096 is not enough

On my laptop, `/proc/stat` is 5200 bytes long and "procs_running" substring is behind 4096's byte from the beginning, so `collectd` filled the logs with "procs_running not found"